web/index.php

```diff
@@ -9,6 +9,8 @@
  * file that was distributed with this source code.
  */
 
+declare(strict_types=1);
```

Comment:
Remove this `declare` from dev-kit files. It should not be introduced with common cs files.